README.md

```diff
@@ -1,41 +1,131 @@
-# ORAS Documentation
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/db61db6e-a953-4b02-b5fb-7f04f018b9d8/deploy-status)](https://app.netlify.com/sites/oras-project/deploys)
+<img src="https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png" alt="FLINT UI logo" height ="auto" width="200" />
+<br />
 
-![ORAS](https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png)
+<h1>ORAS Documentation</h1>
+ 
+An ORAS UI is written in Hugo Framework, to provide an awesome user interface for configuring simulations.
 
-Source for ORAS website and documentation
 
-Link: [oras.land](https://oras.land/)
+<br>
 
-**[The documentation can be found here](/docs/index.md).**
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#developer-notes">Developer notes</a>
+    </li>
+    <li><a href="#how-to-get-involved">How to Get Involved?</a></li>
+    <li><a href="#contributors">Contributors</a></li>
+    <li><a href="#license">Code Of Conduct</a></li>
+  </ol>
+</details>
+<br >
 
-## Deployment
-
-This site is deployed using [Netlify](https://www.netlify.com/)
+## About The Project
+Registries are evolving as generic artifact stores. To enable this goal, the ORAS project provides a way to push and pull OCI Artifacts to and from OCI Registries.
 
+Users seeking a generic registry client can benefit from the ORAS CLI, while developers can build their own clients on top of one of the ORAS client libraries.
 
-## Development
+<br>
 
-System requirements:
+## Getting started
+In order to use and work on this project you need to install python to run the app locally.
 
-- Python 3.7+
+You can install python following the download's [instructions](https://www.python.org/downloads/).
```

Comment:
Might want to still mention minimum Python version people should be using here which right now would be 3.8

Reply:
Okk sir will mention the python version great point make sense to include it